### DISCUSSION TITLE
the application hangs

### DISCUSSION THREAD

**Discussion by NeMysova** [jgraph/drawio#2412] [Q&A]
the application hangs on 6 computers.
Restarting the computer, reinstalling the application does not help, updating the drivers too.
What to do?

**Comment by davidjgraph** [comment]
Please provide environment details, which version was used, etc.

**Reply by NeMysova** [reply]
It's v.15.7.3, win10 and win7

**Reply by davidjgraph** [reply]
And exactly which installer did you use? It's very strange only you get this on 6 computers and there are no other reports. Maybe the downloaded file was corrupted.

**Reply by NeMysova** [reply]
we did not download the same file. There were different, including creating new schemes.
Installer we used in windows distributive, which was collected long time ago. 
we only update the program when it informs about such a need

**Reply by NeMysova** [reply]
everything worked before, problems started months ago or so

**Reply by davidjgraph** [reply]
Uninstall, download 15.7.3 from https://github.com/jgraph/drawio-desktop/releases and install that on each.
